tasks/move-granules/index.js

```diff
@@ -146,7 +145,7 @@ async function moveFileRequest(
   };
 
   // the file moved to destination
-  const fileMoved = clonedeep(file);
+  const fileMoved = { ...file };
```

Comment:
Same note as the other file.

Reply:
Yeah, I checked first and mad sure that this behaves as expected.